### DISCUSSION TITLE
How to deal a large number of small files  in GraphAr ?

### DISCUSSION THREAD

**Discussion by DancyHuang** [apache/incubator-graphar#630] [Ideas]
For some large scale graphs , when use GraphAr as the storage format.  if the underlying storage is HDFS or another object storage system, a large number of small files can have a significant impact on the file system, such as placing considerable pressure on metadata in the case of HDFS. 
Has the Graphr community considered consolidating a large number of small files?

**Comment by SemyonSinchenko** [comment]
We have a goal to create a CLI-tool for working with GraphAr data. It is on roadmap. I think, that `optimize` command that concatenate small files may be a part of such a CLI.
